src/Core/Silk.NET.Core/Loader/DefaultPathResolver.cs

```diff
@@ -260,7 +260,8 @@ private static bool TryLocateNativeAssetFromDeps
                     foreach (var runtimeLib in defaultContext.RuntimeLibraries)
                         foreach (var nativeAsset in runtimeLib.GetRuntimeNativeAssets(defaultContext, rid))
                         {
-                            if (Path.GetFileName(nativeAsset) == name || Path.GetFileNameWithoutExtension(nativeAsset) == name)
+                            var nativeAssetSpan = nativeAsset.AsSpan();
```

Comment:
This API isn't everywhere, can you add an `#ifdef NETSTANDARD2_1`? This should make the build pass :)